cloudhub-2/modules/ROOT/pages/ch2-networking-guide.adoc

```diff
@@ -0,0 +1,196 @@
+
+----
+100.64.0.0/16
+100.66.0.0/16
+100.67.0.0/16
+100.68.0.0/16
+----
+
+Because of this, `100.64.0.0/10` is xref:ps-gather-setup-info.adoc#unusable-cidr[not a supported range] for private spaces.
+
+You can filter these prefixes from the route map. For more informationm see https://help.mulesoft.com/s/article/Received-Extra-BGP-routes-from-CloudHub-2-0-VPN[Received Extra BGP routes from CloudHub 2.0 VPN].
+
+You can also indicate a reserved range for your on-prem network when creating your private space. CloudHub 2.0 doesn't use reserved ranges indicated by customers. For more information, see xref:ps-gather-setup-info.adoc#reserved-cidr[Reserved Corporate CIDR].
+
+=== Connections
+
+Private spaces support two types of connections: Virtual Private Network (VPN) and Transit Gateway (TGW). 
+
+Each Anypoint VPN connection consists of two tunnels that enable you to connect to a single public IP address at a remote location. VPNs are high available if you configure both tunnels on your endpoint. 
+
+Additionally, CloudHub 2.0 enables you to define redundant VPN connections for additional resiliency. As a managed VPN service, the underlying VPN service is frequently upgraded. Routine maintenance can briefly disable one of the two tunnels of your VPN connection. Your VPN connection automatically fails over to the second tunnel during this time, so access is not interrupted. For this reason, you must configure both tunnels on your endpoint. For more information, see  https://help.mulesoft.com/s/article/Tunnel-Switch-in-an-Anypoint-Dynamic-BGP-VPN-Update[Tunnel Switch in an Anypoint Dynamic (BGP) VPN Update]. 
+
+By design, TGW connections are already highly available. Don't create redundant TGW connections. For more information, see xref:ps-tgw-about.adoc[].
+
+==== Connections between CloudHub and CloudHub 2.0
+
+You can use a xref:ps-create-configure-tgw.adoc[TGW attachment] to connect a dedicated VPC in CloudHub to a CloudHub 2.0 private space. You must apply the attachments independently in CloudHub and CloudHub 2.0.
+
+
+=== Firewall Rules and Port Access
+
+HTTP Ingress::
+
+By default, ports `443` and `80` are exposed for all external inbound traffic. You can remove or change these ports to restrict inbound traffic.
+
+
+Non-HTTP Ingress::
+
+By default, no non-http ports are open. You can open a fixed list of TCP ports (30500-32500) to allow non-http traffic to flow to your Mule applications. Those ports are only reachable from within the private space or originated from a VPN or a TGW. For more information about how to allow TCP traffic for your private space, see xref:ch2-deploy-api.adoc[].
+
+
+Egress:: 
+
+By default, applications can make outbound connections to any destination and ports. You can change this behavior to restrict egress traffic.
++
+You can remove all ingress and egress rules from and to the internet. In this case, the cluster still functions normally because of the following control measures:
```

Comment:
"You can remove all ingress and egress rules from and to the internet. In this case, the cluster still functions normally because of the following control measures:"

I think this statement is incorrect. Especially regarding the below bullet point "You can apply egress rules at the application level.".

If customer removes all PS-level egress rules and adds the app-level egress rules, traffic will still fail. When using app-level egress rules, you need to allow the broader traffic at the PS-level and the more specific traffic at the egress-level. It is 2 layers of security rules that the traffic goes through. It needs to be allowed by both the app-level egress rules and also PS-level FW rules.